### ISSUE TITLE
Replace universe:i18n with an npm i18n library

### ISSUE THREAD

**Issue by joshuatshaffer** [Vize-Programming-Team/vize-meteor-app#63]
`universe:i18n` is a Meteor only package. This needs to be replaced before we can leave the Meteor platform. Unfortunately there is no 1-to-1 substitute on NPM so it will probably take some work to migrate our code to a new i18n library.

**Comment by jhigginbotham64** [comment]
A significant part of this is going to be refactoring the code to not make use of the universe/SimpleSchema integration, which is something I'm working on as part of removing SimpleSchema.

**Comment by jhigginbotham64** [comment]
It would be great if someone could find and recommend a good replacement package from NPM. If you want to branch off of goodbye-autoforms and look at the translation refactoring I've already done, I'll be happy to help you start brainstorming ways to actually go about replacing universe:i18n.
